collect_app/src/main/java/org/odk/collect/android/http/OkHttpConnection.java

```diff
@@ -157,7 +157,7 @@ public HttpGetResult executeGetRequest(@NonNull URI uri, @Nullable String conten
             responseHeaders.put(headers.name(i), headers.value(i));
         }
 
-        return new HttpGetResult(downloadStream, responseHeaders, hash, statusCode);
+        return new HttpGetResult(downloadStream, responseHeaders, hash, statusCode, response);
```

Comment:
It feels like an odd design to me that the response is passed to `HttpGetResult` only so it can be closed. The response isn't actually used in `HttpGetResult` so in both cases, it looks like  the response could be closed before the result is returned. Is there a reason you did it this way?